pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3817,17 +3817,13 @@
         "4.3.0"
       ],
       "BaselineVersion": "4.3.0",
-      "InboxOn": {
-        "netcoreapp2.0": "4.3.0.0",
-        "net461": "4.2.0.0",
-        "netstandard2.0": "4.2.0.0",
-        "uap10.1": "4.3.0.0"
-      },
+      "InboxOn": {},
```

Comment:
I suspect we still want this inbox on NETCoreApp since its part of the NETCore.App closure IIRC.  The same is true for UAP.

If you also intend to remove it from the NETCore.App and UAP meta-packages then you should delete the entries in the library's dir.props.

If you don't intend to remove it, then you should add back those inbox entries and make sure that you don't include netcoreapp/uap specific builds in the package.  The validation should lead you down the right path.